common/djangoapps/third_party_auth/tests/specs/test_testshib.py

```diff
@@ -269,8 +269,8 @@ def token_callback(_request, _uri, headers):
             '?$select=username,firstName,lastName,defaultFullName,email'
         )
 
-        def user_callback(_request, _uri, headers):
-            auth_header = headers.get('Authorization')
+        def user_callback(request, _uri, headers):
+            auth_header = request.headers.get('Authorization')
```

Comment:
So, this approach is unfortunately poorly documented by httpretty, so I'm not 100% sure. However, I can tell you that `headers` does not seem to be getting the request headers that are being sent, so I'm trying this out to see if it works better.